packages/twenty-front/src/modules/object-record/record-field/form-types/components/VariableChip.tsx

```diff
@@ -4,10 +4,11 @@ import { searchVariableThroughOutputSchema } from '@/workflow/workflow-variables
 import { css, useTheme } from '@emotion/react';
 import styled from '@emotion/styled';
 import { isDefined } from 'twenty-shared';
-import { IconX } from 'twenty-ui';
+import { IconAlertTriangle, IconX } from 'twenty-ui';
 
-const StyledChip = styled.div<{ deletable: boolean }>`
-  background-color: ${({ theme }) => theme.accent.quaternary};
+const StyledChip = styled.div<{ deletable: boolean; danger: boolean }>`
+  background-color: ${({ theme, danger }) =>
+    danger ? theme.background.danger : theme.accent.quaternary};
   border: 1px solid ${({ theme }) => theme.accent.tertiary};
```

Comment:
We should take care of the border-color in danger mode. According to the Figma, it seems this component should not have any border.

![CleanShot 2025-03-10 at 17 41 11@2x](https://github.com/user-attachments/assets/4635ae05-277d-4b0e-be9c-d73c6a3161bb)

![CleanShot 2025-03-10 at 17 43 17@2x](https://github.com/user-attachments/assets/c091a890-61ff-408d-8532-0f95517f3bab)

I think you can delete this line

```suggestion
```